src/lib/rest/uriParamNames.h

```diff
@@ -45,6 +45,7 @@
 #define URI_PARAM_OPTIONS                 "options"
 #define URI_PARAM_TYPE                    "type"
 #define URI_PARAM_SORTED                  "orderBy"
+#define URI_PARAM_LEVEL                   "level"
```

Comment:
I would add a different section (maybe some neewlines and a new "comment header") for URI params not related with NGSIv2.